addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -241,6 +241,14 @@ const UserValueWidget = Widget.extend({
             this.$el.on('mouseenter.img_animate', 'img', buildImgExtensionSwitcher('png', 'gif'));
             this.$el.on('mouseleave.img_animate', 'img', buildImgExtensionSwitcher('gif', 'png'));
         }
+        this.$el.tooltip({
+            selector: 'we-title',
+            placement: 'bottom',
+            title: function () {
+                const el = this;
+                return (el.scrollWidth > el.offsetWidth) ? el.innerHTML: '';
+            },
+        });
```

Comment:
I asked you to move the code here in the start of UserValueWidget or in the snippets menu itself but actually this is not good in the start here: there are we-title outside of widget. Test your branch and you'll see some label have no tooltips.